webview.h

```diff
@@ -627,6 +628,14 @@ class cocoa_wkwebview_engine {
                    objc_msgSend("NSNumber"_cls, "numberWithBool:"_sel, 1),
                    "developerExtrasEnabled"_str);
     }
+    objc_msgSend(objc_msgSend(config, "preferences"_sel),
+                 "setValue:forKey:"_sel,
+                 objc_msgSend("NSNumber"_cls, "numberWithBool:"_sel, 1),
+                 "javaScriptCanAccessClipboard"_str);
+    objc_msgSend(objc_msgSend(config, "preferences"_sel),
+                 "setValue:forKey:"_sel,
+                 objc_msgSend("NSNumber"_cls, "numberWithBool:"_sel, 1),
+                 "DOMPasteAllowed"_str);
```

Comment:
These are [private APIs](https://github.com/WebKit/webkit/blob/950143da027e80924b4bb86defa8a3f21fd3fb1e/Source/WebKitLegacy/mac/WebView/WebPreferencesPrivate.h#L226) and may result in rejection from the Mac App Store, so we need to figure out another approach

Reply:
Opened #425